overlord/snapstate/snapmgr.go

```diff
@@ -1029,6 +1128,23 @@ func (m *SnapManager) startSnapServices(t *state.Task, _ *tomb.Tomb) error {
 	return err
 }
 
+func scheduleNextRefresh(st *state.State) time.Time {
+	randomness := rand.Int63n(int64(refreshRandomness))
+	nextRefreshTime := time.Now().Add(minRefreshInterval).Add(time.Duration(randomness))
+	st.Set("next-auto-refresh-time", nextRefreshTime)
```

Comment:
Should that be on UTC, so we avoid timezone change and DST pains?

Also, we still need to sync on the way to tweak this (written that just so we can track).